### ISSUE TITLE
Is there a quantization version support in the future? 

### ISSUE THREAD

**Issue by phatjkk** [VinAIResearch/PhoGPT#11]
I hope that PhoGPT will have an AWQ or GPTQ version for running on low VRAM GPUs.
Do you have any plans for quantization? It will make this LLM more popular for students and individual researchers who have limited computing resources.
Thank you for your hard work!

**Comment by datquocnguyen** [comment]
Is there any problem of employing: https://huggingface.co/docs/transformers/quantization#quantization and/or: https://github.com/casper-hansen/AutoAWQ with limited computing resources?

**Comment by phatjkk** [comment]
Yeah I have used AutoAWQ version 0.1.7 (which support Bloom) for quantization PhoGPT-7B5-Instruct to 4 bit GEMM group-size 128 and found this error:

![image](https://github.com/VinAIResearch/PhoGPT/assets/48487157/633f4075-8246-45f8-afd5-dc2799fe68e0)

The AutoAWQ working well with vietcuna-7b-v3 (which used Bloom too).

Here is the code I have used:
https://github.com/phatjkk/test_llm/blob/main/FINALL_GENERATOR_Quantization.ipynb

I don't know what the problem is. I tried some previous version of transformer but may not work!
Are you try it yet?
Thank you!
